python/paddle/fluid/tests/unittests/test_is_tensor_op.py

```diff
@@ -0,0 +1,56 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import paddle
+
+DELTA = 0.00001
+
+
+class Test_is_tensor_op(unittest.TestCase):
+    def test_is_tensor_real(self, dtype="float32"):
+        """Test is_tensor api with a real tensor
+        """
+        x = paddle.rand([3, 2, 4], dtype=dtype)
+        res_expect = True
```

Comment:
You can make code shorter in the three test cases here:
```
self.assertTrue(paddle.is_tensor(x))
```

You don't need `res_expect` because `assert` means that we expect the result will be xxx

Reply:
Modified.